src/GslCore/DnaCreation.fs

```diff
@@ -552,7 +536,29 @@ let expandAssembly
 
                 match ppp.part with
                 | MARKERPART ->
-                    yield expandMarkerPart library dnaSource ppp
+                    // Choose provider
+                    let providers =
+                        markerProviders |>

```

Comment:
Please normalize the indentation of this block and move the pipe operator to the beginning of the line.